cypress/e2e/shared/flowsAlerts.test.ts

```diff
@@ -0,0 +1,429 @@
+import {Organization} from '../../src/types'
+
+describe('flows alert panel', () => {
+  beforeEach(() =>
+    cy.flush().then(() =>
+      cy.signin().then(() =>
+        cy.get('@org').then(({id}: Organization) =>
+          cy.fixture('routes').then(({orgs}) => {
+            cy.visit(`${orgs}/${id}`)
+            cy.getByTestID('version-info')
+            return cy
+              .setFeatureFlags({
+                simpleTable: true,
```

Comment:
do we still need the simpleTable flag or has that been changed to a different flag name?

Reply:
good catch - we don't need this anymore